CONTRIBUTING.md

```diff
@@ -19,6 +19,11 @@ We are excited to work with the open source communities in the many years to com
 - If it is appropriate to write a design document, the document must be hosted either in the GitHub tracking issue, or linked to from the issue and hosted in a world-readable location.
 - Specifically, if the goal is to add a new extension, please read the extension policy.
 - Small patches and bug fixes don't need prior communication. If you have identified a bug and have ways to solve it, please create an [issue](https://github.com/unitycatalog/unitycatalog/issues) or create a [pull request](https://github.com/unitycatalog/unitycatalog/pulls).
+- In this project, only maintainers can start [GitHub Discussions](https://github.com/unitycatalog/unitycatalog/discussions).
+- Non-maintainers who wish to initiate a Discussion typically follow one of two approaches:  
+  1. Create an issue and ask a maintainer to convert it to a Discussion.  
+  2. Contact a maintainer on Slack to request the creation of a Discussion.
```

Comment:
New developers don't know who the maintainer is. We should document how to find maintainers on Slack.

Reply:
Hi @ebyhr - this is a good call out.  Part of the reason for the delay is due to finalizing the LF contracts.  Saying this, we're pretty close to getting this resolved (admittedly, this has been going on for about 9 months longer than expected).  Once  this is done, we can start adding maintainers to the projec.t

Reply:
Oh, and to add to this - perhaps we can add a call out to ping the `#contributors` channel to contact the maintainers?  WDYT @ebyhr @ognis1205

Reply:
I see the activity in `#contributors` isn't so active. Just mentioning the channel name looks sufficient for now.